src/constants/translations/en/student-home-page.json

```diff
@@ -1,8 +1,11 @@
 {
   "findTutorBlock": {
-    "title": "A Global Network of Tutors",
+    "title": {
+      "student": "A Global Network of Tutors",
+      "tutor": "A Global Network of Students"
```

Comment:
Add translations for the Ukrainian language

Reply:
There are no student-home-page.json for ua folder to change it

Reply:
Create a file and add not everything, but at least this. 
It is the worst practice that doesn't do it.
In the future, we face problems when implementing language toggle. And it will need to test everything and add the Ukrainian language where it is needed.

Reply:
Understood👍